### ISSUE TITLE
overfitting ?

### ISSUE THREAD

**Issue by Rexzhan** [junjun-jiang/SSPSR#1]
（chikusei）论文里说的32batchsize60epoch，但是我训练的时候在过了20-30epoch时性能已经开始下降了呢，val loss 也已经稳定， 是网络已经过拟合了吗

**Comment by junjun-jiang** [comment]
我这边训练的时候没有出现你说的问题。

**Comment by chenjiachengzzz** [comment]
> 
您好，请问您的数据是怎么组织的，能分享一下目录嘛

**Comment by chenjiachengzzz** [comment]
> （chikusei）论文里说的32batchsize60epoch，但是我训练的时候在过了20-30epoch时性能已经开始下降了呢，val loss 也已经稳定， 是网络已经过拟合了吗

您好，请问您的数据是怎么组织的，能分享一下目录嘛
